frontend/sozisel-app/src/components/PresenterSession/EventsTimline/QuizDetails/QuizDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import "./QuizDetails.scss";
+
+import React, { ReactElement } from "react";
+
+import { Quiz } from "../../../../model/Template";
+import { useTranslation } from "react-i18next";
+
+interface QuizDetailsProps {
+  quiz: Quiz;
+}
+
+export default function QuizDetails({ quiz }: QuizDetailsProps): ReactElement {
+  const { t } = useTranslation("common");
+  return (
+    <div className="QuizDetails">
+      <div>
+        <p>
+          {t(
+            "components.PresenterSession.EventsTimeline.QuizDetails.durationTime",
+            { value: quiz.durationTimeSec }
+          )}
+        </p>
+        <p>
+          {t(
+            "components.PresenterSession.EventsTimeline.QuizDetails.percentageOfParticipants",
+            { value: quiz.targetPercentageOfParticipants }
+          )}
+        </p>
+        <p>
+          {quiz.trackingMode
+            ? t(
```

Comment:
what about 
```js
const trackingModeLabel = quiz.trackingMode ? "enableTrackingMode" : "disableTrackingMode";

return (...
t(`components.PresenterSession.EventsTimeline.QuizDetails.${trackingModeLabel}`)
```

Reply:
I tried in this way and some other way, but every time `npm run format `changes it.